components/eam/src/chemistry/mozart/mo_gas_phase_chemdr.F90

```diff
@@ -48,8 +48,8 @@ module mo_gas_phase_chemdr
   character(len=fieldname_len),dimension(extcnt)        :: extfrc_name
   logical :: convproc_do_aer 
 
-  real(r8), parameter :: low_limit = 0.1_r8 ! lower limit factor for dry deposition
-  real(r8), parameter :: up_limit  = 2.0_r8 ! upper limit factor for surface emission
+  real(r8), parameter :: low_limit = 0.1_r8  ! lower limit factor for dry deposition
+  integer,  parameter :: srf_emit_nlayer = 3 ! number of layers adding surface emission
```

Comment:
What's the point of this?
Is there not enough diffusion in the boundary layer?

Reply:
The up-limit is not used, should have been deleted...will fix it.
In our previous test, dumping all emissions in the single layer could cause issues.

Emitting to the bottom 3 layers in my view is to provide a more realistic initial guess for the diffusion to act on. In case diffusion is not enough, it avoids extreme concentrations for chemistry solvers.

Hui Wan has proposed a change to the order of physics processes, which is being considered by NGD and it should further improve emissions.

Reply:
I don't understand what you mean by "initial guess". Smearing the emissions across the layers is effectively adding extra diffusion. 

Are these emissions being added in tphysac? If so, then I would guess the problems you're referring to are because CLUBB doesn't do any diffusion until tphysbc. If that's the case then I suppose adding the "psuedo" diffusion here is warranted, but seems very "hacky". If I were in charge of this I would want to consider adding an additional tracer diffusion calculation to tphysac to avoid this sort of workaround (like we do for the MMF). 

Does Hui's proposed process ordering include putting a diffusive calculation in tphysac?

Reply:
It's a matter of how to specify the surface emission for the diffusion to act on. If we think the state after diffusion "the solution", then before it is "the guess".

In the real continuous world, surface emission occurs at the surface. In the modeled discrete world, that's not necessarily true. I'd argue if the diffusion is handled robustly, it shouldn't be too sensitive to these two ways of adding emissions at least for chemistry.

It is in the chemistry - I am not sure about tphysac.